### PR TITLE
Add linting package to `cloud-functions` directory

### PR DESCRIPTION
**Pull request by tarikbellamine**
### Description
Add linting package to the `cloud-functions` directory and fix errors.

### Other changes
Updated `CODEOWNERS` to reflect this new repo

### Tested
Yes

### How others should test
N/A

### Related issues
N/A

### Backwards compatibility
Yes

### CODE REVIEW

**Review by silasbw — APPROVED** [review]
the eslint setup lgtm

File: packages/cloud-functions/tsconfig.json [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
+    "strictNullChecks": true,
+    "skipLibCheck": true,
+    "target": "es2015",
+    "strict": true
   },
-  "include": ["src", "index.d.ts"],
-  "compileOnSave": true
+  "include": ["src"],
+  "exclude": ["node_modules", "vendor", ".bundle"],
+  "references": [{ "path": "../komencikit" }]
```
Comment (jeanregisser) [inline packages/cloud-functions/tsconfig.json:22]: Why is this needed?

Comment (jeanregisser) [reply packages/cloud-functions/tsconfig.json:22]: No problem deploying with these changes?

**Review by jeanregisser — COMMENTED** [review]
Awesome 👍

File: packages/cloud-functions/tsconfig.json (hunk 1 above)
Comment (tarikbellamine) [reply packages/cloud-functions/tsconfig.json:22]: That was a leftover from the `mobile` config that's not needed here. Deleted it.

No problems building and deploying!
